.gitignore

```diff
@@ -0,0 +1,3 @@
+examples/cloudwatch/init.md
+examples/cloudwatch/terraform.tfvars
+examples/cloudwatch/README.md
```

Comment:
README.md is good to push.

Reply:
Besides that, please place the `.gitignore` file in the `examples/cloudwatch` directory.